index.html

```diff
@@ -6,17 +6,249 @@
       name="viewport"
       content="width=device-width, initial-scale=1.0"
     />
-    <title>Miami</title>
+    <title>My-Bike</title>
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com"
+    crossorigin="anonymous">
+    <link href="https://fonts.googleapis.com/css2?family=Poppins:wght@400;700&display=swap" rel="stylesheet">
     <link
       rel="stylesheet"
-      href="src/styles/main.scss"
+      href="src/styles/main.css"
     />
     <script
       src="src/scripts/main.js"
       defer
     ></script>
   </head>
   <body>
-    <h1>Miami</h1>
+
+    <header class="header" id="home">
+      <nav class="header__nav">
+
+        <h1 class="header__nav--logo">MyBiKE</h1>
```

Comment:
`--` is used for modifiers not elements and modifiers should always be [used together with their belonging class](https://mate-academy.github.io/fe-program/css/typical-bem-mistakes-en#using-a-modifier-without-the-belonging-class)

you can apply one of the following conventions:
- replace all `--` with `-`
```suggestion
        <h1 class="header__nav-logo">MyBiKE</h1>
```
- keep only the part after `--` 
```suggestion
        <h1 class="header__logo">MyBiKE</h1>
```
-  create separate `nav` block
```suggestion
    <header class="header" id="home">
      <nav class="header__nav nav">
        <h1 class="nav__logo">MyBiKE</h1>
```